donkeycar/management/kivy_ui.py

```diff
@@ -856,16 +861,24 @@ def on_aug_list(self, obj, aug_list):
         if not self.config:
             return
         self.config.AUGMENTATIONS = self.aug_list
-        self.augmentation = ImageAugmentation(self.config, 'AUGMENTATIONS',
-                                              always_apply=True)
+        self.augmentation = ImageAugmentation(
+            self.config, 'AUGMENTATIONS', always_apply=True)
         self.on_current_record(None, self.current_record)
 
     def on_trans_list(self, obj, trans_list):
         if not self.config:
             return
         self.config.TRANSFORMATIONS = self.trans_list
-        self.transformation = ImageAugmentation(self.config, 'TRANSFORMATIONS',
-                                                always_apply=True)
+        self.transformation = ImageTransformations(
+            self.config, 'TRANSFORMATIONS')
+        self.on_current_record(None, self.current_record)
+
+    def on_post_trans_list(self, obj, trans_list):
```

Comment:
Ok, I see what is going on.  Yes, then we can change the ImageTransformations to take the config name.  I can change the inference to use two parts rather than one.

Reply:
I think it's fine to have one part that uses both lists together.